src/client/pages/todos.react.js

```diff
@@ -1,15 +1,12 @@
-import Buttons from './buttons.react';
+import React from 'react';
```

Comment:
Ah, all righty. I am always sorting by `dependencies` first, then internal files. Didn't know you sort by name. Will do!